tools/manually-handling-sharding-ddl-locks.md

```diff
@@ -0,0 +1,415 @@
+            ],
+            "synced": [                                    # the list of DM-workers that have received all sharding DDL events in the corresponding MySQL instances
+                "127.0.0.1:8262"
+            ],
+            "unsynced": [                                  # the list of DM-workers that have not yet received all sharding DDL events in the corresponding MySQL instances
+                "127.0.0.1:8263"
+            ]
+        }
+    ]
+}
+```
+
+#### unlock-ddl-lock
+
+Actively ask `DM-master` to unlock the specified DDL lock, for example, ask the owner to execute the DDL; ask other DM-workers that are not the owner to execute or skip the DDL; remove the lock information on `DM-master`.
+
+##### Command usage
+
+```bash
+unlock-ddl-lock [--worker=127.0.0.1:8262] [--owner] [--force-remove] <lock-ID>
+```
+
+##### Variables description
+
+- `worker`: flag; string; `--worker`; optional; can be specified multiple times; if not being specified, send requests for all DM-workers that are waiting for the lock to skip the DDL; if being specified, send requests for the specified DM-worker to skip the DDL
+- `owner`: flag; string; `--owner`; optional; if not being specified, request for the default owner (the owner in the result of `show-ddl-locks`) to execute the DDL; if being specified, request for the DM-worker (the alternative of the default owner) to execute the DDL 
+- `force-remove`: flag; boolean; `--force-remove`; optional; if not being specified, remove the lock information only when the owner succeeds to execute the DDL; if being specified, compulsorily remove the lock information even though the owner fails to execute the DDL (after doing this you will not be able to query or operate on the lock again)
+- `lock-ID`: non-flag; string; optional; specify the ID of the DDL lock which needs to be unlocked (the `ID` in the result of `show-ddl-locks`)
+
+##### Example of results
+
+```bash
+» unlock-ddl-lock test-`shard_db`.`shard_table`
+{
+    "result": true,                                        # show if the unlocking process succeeds
+    "msg": "",                                             # show the reason why the unlocking process fails
+    "workers": [                                           # the result list of the DDL execution/skipping operation of each DM-worker
+        {
+            "result": true,                                # show if the DDL execution/skipping operation of the DM-worker succeeds
+            "worker": "127.0.0.1:8262",                    # the address of the DM-worker (DM-worker ID)
+            "msg": ""                                      # show the reason why the DDL execution/skipping operation of the DM-worker fails
+        }
+    ]
+}
+```
+
+#### break-ddl-lock
+
+Actively ask the DM-worker to compulsorily break the DDL lock that is waiting to be unlocked, for example, ask the DM-worker to execute/skip the DDL; remove the DDL lock information on the DM-worker.
```

Comment:
I have a problem: what's differences betwen `break-ddl-lock` and `unlock-ddl-lock` according to your understanding?  @yikeke